test/run-test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e 	# Exit immediately upon failure
+set -o pipefail  # Carry failures over pipes
+
+repo_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+docker_repo="microsoft/dotnet-nightly"
+
+if [ -z "${DEBUGTEST}" ]; then
+    optional_docker_run_args="--rm"
+fi
+
+for cli_channel in $( find . -mindepth 1 -maxdepth 1 -type d ! -name .git ! -name test ! -name update-dependencies -print | sed -e 's/\.\///' ); do
```

Comment:
Good idea.

---
In reply to: [76322475](https://github.com/dotnet/dotnet-docker-nightly/pull/35#discussion_r76322475) [](ancestors = 76322475)